test/rules/beta/test_out.jl

```diff
@@ -0,0 +1,30 @@
+module RulesBetaOutTest
+
+using Test
+using ReactiveMP
+using Random
+import ReactiveMP: @test_rules
+
+@testset "rules:Beta:out" begin
+
+     @testset "Belief Propagation: (m_a::PointMass, m_b::PointMass)" begin
+
+        @test_rules [ with_float_conversions = true ] Beta(:out, Marginalisation) [
+            (input = (m_a = PointMass(1.0), m_b = PointMass(2.0)), output = Beta(1.0, 2.0)),
+            (input = (m_a = PointMass(2.0), m_b = PointMass(2.0)), output = Beta(2.0, 2.0)),
+            (input = (m_a = PointMass(3.0), m_b = PointMass(3.0)), output = Beta(3.0, 3.0)),
+        ]
+
+    end
+
+    @testset "Belief Propagation: (q_a::PointMass, q_b::PointMass)" begin
```

Comment:
@A-Vzer  Rules that use `q_` variables in it are **Variational Message Passing** rules, not Belief Propagation. Could you please change it everywhere? If all inputs are `m_` than it is the **Belief Propagation**